src/client/src/ui/spotTile/components/notifications/NotificationContainer.tsx

```diff
@@ -34,7 +34,7 @@ export default class NotificationContainer extends PureComponent<Props> {
       )
     }
     if (!lastTradeExecutionStatus || !hasNotification(lastTradeExecutionStatus)) {
-      return () => null
+      return () => null as SFC<void>
```

Comment:
Is this needed?

Reply:
Yes => returning null is inferred as any